bootstrap/standalone/src/main/java/org/geysermc/geyser/platform/standalone/GeyserStandaloneBootstrap.java

```diff
@@ -258,7 +262,7 @@ private boolean isHeadless() {
     @Override
     public void onGeyserDisable() {
         // We can re-register commands on standalone, so why not
-        GeyserImpl.getInstance().commandManager().getCommands().clear();
+        commandRegistry.commands().clear();
```

Comment:
i'll probably just remove this because the registry is recreated during enable/reload

Reply:
On standalone  (and some others), it is created during onGeyserEnable.

Reply:
well, that's the issue
On ViaProxy it is possible to start the program - which will initialize Geyser - but won't start it until the user actually starts the proxy. If that never happens, this will throw